packages/fast-tooling-react/src/form/templates/plugin.control.abstract.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {
+    AbstractControlTemplateProps,
+    ControlConfig,
+} from "./template.control.abstract.props";
+
```

Comment:
Does "Abstract" really need to be in this name? We typically avoid naming conventions like this (I for interface, for example)

Reply:
Do you have a preference? I could go with "Common" or "Base".